launch/tier4_system_launch/launch/system.launch.xml

```diff
@@ -126,13 +102,8 @@
       </include>
     </group>
 
-    <!-- Hazard Status Converter -->
-    <group unless="$(var use_emergency_handler)">
```

Comment:
The original logic appears to be reversed since the converter is needed when mrm_handler is used.
Leave it without the if statement.

Reply:
It's "unless", so the logic was correct. remove the if statement.

Reply:
Thank you 👍 

fixed in https://github.com/autowarefoundation/autoware.universe/pull/8917/commits/f2f69fcbdc287a0d4ceaa47c7204267826347b5a